frontend/public/components/pod-exec.jsx

```diff
@@ -47,10 +47,16 @@ export const PodExec = connectToFlags(FLAGS.OPENSHIFT)(
     }
 
     connect_() {
-      const { metadata } = this.props.obj;
+      const {
+        metadata,
+        spec: { tolerations },
+      } = this.props.obj;
       const { activeContainer } = this.state;
       const usedClient = this.props.flags[FLAGS.OPENSHIFT] ? 'oc' : 'kubectl';
-
+      const isWindows = _.some(tolerations, (t) => {
+        return t.key === 'os' && t.value === 'Windows';
+      });
```

Comment:
nit: Prefer the native `Array.some`
```suggestion
      const isWindows = tolerations?.some((t) => {
        return t.key === 'os' && t.value === 'Windows';
      });
```